homeassistant/components/gogogate2/cover.py

```diff
@@ -96,7 +98,7 @@ def __init__(
     @property
     def available(self) -> bool:
         """Return True if entity is available."""
-        return self._is_available
+        return self._data_update_coordinator.last_update_success
 
```

Comment:
You can drop `available` and `should_poll` now since they are provided by `CoordinatorEntity`